src/components/fieldset/index.md.njk

```diff
@@ -31,4 +31,4 @@ Read more about [why and how to set legends as headings](/get-started/labels-leg
 
 On [question pages](/patterns/question-pages/) containing a group of inputs, including the question as the legend helps users of screen readers to understand that the inputs are all related to that&nbsp;question.
 
-Include any general help text which is important for filling in the form and cannot be written as [hint text](/components/text-input/#hint-text) in the legend, but try to keep it as concise as possible.
+Include general help text in the legend if it would help the user fill in the form, and you cannot write it as [hint text](/components/text-input/#hint-text). However, try to keep it as short as possible.
```

Comment:
@calvin-lau-sig7 Joe commented in the original issue that an example might be good under here. What do you think?

Reply:
Hi @joelanman, when you have a chance, can we have a chat so I can understand what the situation for this might be?

Reply:
We're going to release, happy to add an example as a further PR.